tests/bootstrap.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Is this file actually needed, considering composer's autoloader?

Reply:
yes it is, see https://github.com/slevomat/coding-standard/blob/1ff7cfb8e8df3af138cd04793ca1fbfea254cd50/phpunit.xml.dist#L3

Code_sniffer has custom auto loading on tests